$OpenBSD: patch-Makefile,v 1.62 2016/04/02 15:03:37 benoit Exp $

No need for librt on OpenBSD

Workaround for ar(1) not having 's' on VAX by calling ranlib

--- Makefile.orig	Mon Mar 28 22:17:31 2016
+++ Makefile	Sat Apr  2 16:49:57 2016
@@ -1471,7 +1471,6 @@ endif
 
 ifdef HAVE_CLOCK_GETTIME
 	BASIC_CFLAGS += -DHAVE_CLOCK_GETTIME
-	EXTLIBS += -lrt
 endif
 
 ifdef HAVE_CLOCK_MONOTONIC
@@ -2032,13 +2031,13 @@ $(REMOTE_CURL_PRIMARY): remote-curl.o http.o http-walk
 		$(CURL_LIBCURL) $(EXPAT_LIBEXPAT) $(LIBS)
 
 $(LIB_FILE): $(LIB_OBJS)
-	$(QUIET_AR)$(RM) $@ && $(AR) $(ARFLAGS) $@ $^
+	$(QUIET_AR)$(RM) $@ && $(AR) rc $@ $^ && ranlib $@
 
 $(XDIFF_LIB): $(XDIFF_OBJS)
-	$(QUIET_AR)$(RM) $@ && $(AR) $(ARFLAGS) $@ $^
+	$(QUIET_AR)$(RM) $@ && $(AR) rc $@ $^ && ranlib $@
 
 $(VCSSVN_LIB): $(VCSSVN_OBJS)
-	$(QUIET_AR)$(RM) $@ && $(AR) $(ARFLAGS) $@ $^
+	$(QUIET_AR)$(RM) $@ && $(AR) rc $@ $^ && ranlib $@
 
 export DEFAULT_EDITOR DEFAULT_PAGER
 
@@ -2223,7 +2222,7 @@ export TEST_NO_MALLOC_CHECK
 
 ### Testing rules
 
-test: all
+test:
 	$(MAKE) -C t/ all
 
 perf: all
